### ISSUE TITLE
Orderable Updates Position when Destroying Parent Embedded In

### ISSUE THREAD

**Issue by brand-it** [mongoid/mongoid_orderable#45]
When you call something like `Grouping.find(1234).destroy` the cascading call update the children position in the database rather then just destroying the parent data and moving along. 

``` ruby
class Group
  include Mongoid::Document
  include Mongoid::Timestamps

  embeds_many :items, cascade_callbacks: true
end
class Item
  include Mongoid::Document
  include Mongoid::Timestamps
  include Mongoid::Orderable

  orderable

  embedded_in :group
end
```

This is not a the worst that can happen however it does slow down the destroy_all method and destroy method. It was a bit of extra work to track down but  you can see it updating the position in the log file. One of the requirements is that you have to have more than one item embedded in the grouping but if you do it will result in the position field -1 for every nested item within the grouping excluding the first item that you destroyed.

**Comment by pyromaniac** [comment]
Hey!
> One of the requirements is that you have to have more than one item embedded in the grouping
Where is ite requirement from?

Also I'm not sure that I'll be able to solve this issue soon. If you will submit a PR with the fix - I'll merge it right away.

**Comment by newdark** [comment]
Yeah I am going to see about working on a fix for this. It has not been a super big problem because we we don't destroy data in most of the apps that use this gem. However when we do it does slow the system down by about 3x to nothing at all. It really fluctuates depending on the situation. I hope this helps someone else out that is looking into this bug.

**Comment by johnnyshields** [comment]
Will be fixed in version 6.0.0
